.github/workflows/ci.yml

```diff
@@ -16,9 +16,11 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          # Earliest-supported versions.
-          - elixir: "1.7.4"
-            otp: "21.3.8.17"
+          # Earliest OTP version available on ubuntu-22.04 with compatible Elixir.
+          # Broadway requires Elixir ~> 1.7, but OTP 24.2+ is the minimum on ubuntu-22.04.
+          # Elixir 1.12 is the earliest version compatible with OTP 24.
+          - elixir: "1.12"
+            otp: "24.2"
```

Comment:
@josevalim can you help me here? What should I do? What matrix including the Ubuntu version so should I use?

Reply:
Yes, it is fine to bump the minimum Elixir version. I did so in other projects too. Whatever is minimum that works.